helm/bits/templates/bits-config.yaml

```diff
@@ -50,7 +50,7 @@ stringData:
         private_endpoint: https://{{ .Values.blobstore.serviceName }}.{{ .Release.Namespace }}.svc.cluster.local:4443
         {{- if .Values.ingress.use }}
         public_endpoint: https://blobstore.{{ .Values.ingress.endpoint }}
-        {{- else if .Values.services.loadbalanced }}
+        {{- else if .Values.services.nodePort }}
         public_endpoint: https://blobstore.{{ .Values.env.DOMAIN }}
```

Comment:
That seems odd: if we're using nodeport, surely the port should be in there somewhere? (Yes that means that we will need to adapt the kubecf part to remove some of the hacks.)

Reply:
Hm ... Oh. Yes, we currently have the port info in kubecf, see https://github.com/cloudfoundry-incubator/kubecf/blob/56f87f06fdedf93929d43f196af36084bc48be5f/deploy/helm/kubecf/values.yaml#L373-L376

Will fix, i.e. shift the part from kubecf to here.

Reply:
Done with commit 1e05c56e27fc4fd4b7d4d3ac2226306029f0f315
and https://github.com/cloudfoundry-incubator/kubecf/pull/1022/commits/55ab51b135d41881af2cc8141cbd00e147e564e9